setup.py

```diff
@@ -132,8 +134,11 @@ def get_extensions():
     if has_ffmpeg:
         ffmpeg_bin = os.path.dirname(ffmpeg_exe)
         ffmpeg_root = os.path.dirname(ffmpeg_bin)
-        ffmpeg_include_dir = os.path.join(ffmpeg_root, 'include')
-
+        include_dir = os.path.join(ffmpeg_root, 'include')
+        ffmpeg_include_dir = include_dir
+        include_subdir = os.path.join(include_dir, 'ffmpeg')
```

Comment:
So `libavutil` etc are under a `ffmpeg` folder?
Hum, given that we don't currently have CI for Fedora, I'm afraid that this will get broken very quickly, and we might need to find a more principled way of getting the include dirs for the ffmpeg libraries.

Do you have some ideas on how to get this in a more reliable manner?

Reply:
Well, I think the way /usr (or /usr/local) path is found is OK, now for the {prefix}/include dir I don't see many options. Maybe searching for the header file (from inside {prefix}/include) ? Or just include both folders ({prefix}/include and {prefix}/include/ffmpeg (if the later exists).

Reply:
edit: I pushed an update with a dummy search for the header file, it seems to be working, and does not take a lot of time.